gdx/src/com/badlogic/gdx/graphics/Texture.java

```diff
@@ -124,6 +124,14 @@ public Texture (TextureData data) {
 		load(data);
 		if (data.isManaged()) addManagedTexture(Gdx.app, this);
 	}
+
+	public Texture (TextureData data, int glHandle) {
+		super(GL20.GL_TEXTURE_2D, glHandle);
+		load(data);
+		if (data.isManaged()) {
+			addManagedTexture(Gdx.app, this);

```

Comment:
Depends on the use-case. For LwjglGL30 (probably any device that supports gles3) it can be safely ignored afaik, but this api is not specifc to lwjgl or gles3. Perhaps make this constructor protected and add a javadoc warning to override the reload method as well.